synapse/handlers/register.py

```diff
@@ -531,3 +533,15 @@ def _join_user_to_room(self, requester, room_identifier):
             remote_room_hosts=remote_room_hosts,
             action="join",
         )
+
+    def _check_mau_limits(self):
```

Comment:
besides the same comment as above you might want to deduplicate this somehow

Reply:
same as above, and also the calculating the total count is duped with the daily count - but likley to change structure